test/src/analyzers/lint_analyzer/rules/rules_list/check_for_equals_in_render_object_setters/examples/example.dart

```diff
@@ -0,0 +1,77 @@
+class RenderObject {}
+
+class RenderBox extends RenderObject {}
+
+class SomeRenderBox extends RenderBox {
+  int _page;
+  int get page => _page;
+  set page(int value) {
+    if (value == _page) {
+      return;
+    }
+    _page = value;
+    markNeedsLayout();
+  }
+
+  double get overflowSpacing => _overflowSpacing;
+  double _overflowSpacing;
+  set overflowSpacing(double value) {
+    assert(value != null);
+    if (_overflowSpacing == value) return;
```

Comment:
IIRC there is also a case:

```dart
if (_overflowSpacing != value) {
  _overflowSpacing = value;
  markNeedsLayout()
}
```

But forget whether this is rare or not. Maybe we can run it on flutter framework repo first and see

Reply:
Hm, thanks, looks valid

Reply:
Fixed, ~10 false-positives left, but I think they're mostly for manual ignore